.travis.yml

```diff
@@ -71,6 +71,7 @@ before_install:
 
     # Cozy stack v3
     - travis_retry go get -u github.com/cozy/cozy-stack
+    - cp ./dev/cozy-stack/cozy.yaml $HOME/cozy.yaml
     - $GOPATH/bin/cozy-stack serve --log-level warning >/dev/null &
```

Comment:
Maybe you can use `--fs-url` here or move the log level option to the config file. It looks weird to have configuration from both the command-line flags and the config file.